<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="text">
          <string>Image-To-Icon</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>117</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_convert">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Convert</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>117</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuOutput_Path">
     <property name="title">
      <string>Output_Path</string>
     </property>
     <addaction name="SameLocation"/>
     <addaction name="separator"/>
     <addaction name="SelectLocation"/>
    </widget>
    <addaction name="menuOutput_Path"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuContact">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Contact</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionYouTube"/>
    <addaction name="separator"/>
    <addaction name="actionGitHub"/>
    <addaction name="separator"/>
    <addaction name="actionFacebook"/>
    <addaction name="separator"/>
    <addaction name="actionInstagram"/>
    <addaction name="separator"/>
    <addaction name="actionTelegram"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuContact"/>
  </widget>
  <action name="SameLocation">
   <property name="text">
    <string>Same Location</string>
   </property>
  </action>
  <action name="SelectLocation">
   <property name="text">
    <string>Select Location</string>
   </property>
  </action>
  <action name="actionYouTube">
   <property name="text">
    <string>YouTube</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="actionFacebook">
   <property name="text">
    <string>Facebook</string>
   </property>
  </action>
  <action name="actionInstagram">
   <property name="text">
    <string>Instagram</string>
   </property>
  </action>
  <action name="actionTelegram">
   <property name="text">
    <string>Telegram</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
